### DISCUSSION TITLE
redirect authentificated users to different pages by role using breeze authentification

### DISCUSSION THREAD

**Discussion by manelbenmiled** [spatie/laravel-permission#2198] [Q&A]
hi,
i want to redirect users authentificated to specific pages after login 

**NB**: i assign role to every user that is connected and i'm using Active Directory to login 

**LoginController:**
`protected $redirectTo = RouteServiceProvider::HOME;`

**RouteServiceProvider**
`public const HOME='/';`

How can i set up this please?

**Comment by parallels999** [comment]
That is a question for laravel framework, read laravel's documentation
